Dockerfile

```diff
@@ -62,6 +63,30 @@ RUN pip3 install --no-cache-dir -r ./docs/requirements.txt
 COPY . /code/listenbrainz
 
 
+#########################################################################
+# NOTE: The javascript files for production are compiled in this image. #
+#########################################################################
+FROM node:$NODE_VERSION as listenbrainz-frontend
```

Comment:
This seems to be basically equivalent to the Dockerfile.webpack file.
Do we still need that separate dockerfile? or can we change the static_builder in docker-compose to use `target: listenbrainz-frontend` ?

Reply:
Hmm, good idea let me see if I can make that work.